### ISSUE TITLE
[no-import-module-export] crashes because of not being able to find module

### ISSUE THREAD

**Issue by ljosberinn** [benmosher/eslint-plugin-import#2062]
Error:
![image](https://user-images.githubusercontent.com/29307652/118273425-30c8fc80-b4c4-11eb-9c7c-6d3228f3df3a.png)

Logging within getEntryPoint within the rule:
![image](https://user-images.githubusercontent.com/29307652/118274060-f875ee00-b4c4-11eb-9292-6222ae10c915.png)

```js
function getEntryPoint(context) {
  const pkgPath = _pkgUp2.default.sync(context.getFilename());
  console.log({context, contextFileName: context.getFilename(), pkgPath,dirName: _path2.default.dirname(pkgPath)})

  return require.resolve(_path2.default.dirname(pkgPath));
}
```
looks like its trying to retrieve the package json but from the wrong folder?

**Comment by ljosberinn** [comment]
false positive, forgot I have a test-specific package.json in that folder..
